### DISCUSSION TITLE
Redirect

### DISCUSSION THREAD

**Discussion by Shizanit** [YOURLS/YOURLS#3320] [Q&A] [answered]
advise how to do not embed, but redirect to the link

![image](https://user-images.githubusercontent.com/65107731/166429114-f5644dc5-1f63-4f50-8f09-9043b1804c41.png)

**Comment by dgw** [comment, marked as answer]
Deactivate the [bundled YOURLS Toolbar plugin](https://github.com/YOURLS/YOURLS/blob/master/user/plugins/sample-toolbar/plugin.php) you apparently turned on.
